toil_orchestrator/models.py

```diff
@@ -26,7 +26,6 @@ class Job(BaseModel):
     root_dir = models.CharField(max_length=1000)
     job_store_location = models.CharField(max_length=1000, null=True, blank=True)
     working_dir = models.CharField(max_length=1000, null=True, blank=True)
-    output_dir = models.CharField(max_length=1000, null=True, blank=True)
```

Comment:
You didn't commit migration. If you are changing model migrations should also be included in PR.

Reply:
Since it is already merged, you can make a separate PR. Everything else looks good.

Reply:
you are right, I created a new PR